package.json

```diff
@@ -6,10 +6,9 @@
   "scripts": {
     "dev": "node server.js",
     "build": "next build",
-    "start": "NODE_ENV=production node server.js",
     "test": "jest",
-    "prettier-js": "prettier --write '{pages,src}/**/*.{js,jsx}'",
-    "prettier-md": "prettier --write 'static/docs/**/*.md'",
+    "start": "NODE_ENV=production node server.js",
+    "pretty": "pretty-quick --staged --no-restage --bail",
     "lint": "yarn run eslint src"
```

Comment:
As noted in https://github.com/iterative/dvc.org/pull/647#issuecomment-538110156, I realized we have 2 scripts for similar purposes: `yarn pretty` (uses `pretty-quick`) and `yarn lint` (uses `eslint`). Both are enforced locally with a Git pre-commit hook (see line 71 of this same file) but this could be disabled by the developer. CircleCI only enforces the latter, `yarn lint`.

Is this correct? Seems strange to me @shcheklein and @iAdramelk.

---

p.s. I cleaned up some old linting scripts as you can see in e74cbc1.

Reply:
Yes, we should add prettier to the CI too. But it should be validate-only task, without attempts to fix code like it pre-commit hooks.

Reply:
OK, changed in 0e82cd7 can you please check? Seems to work based on [the corresponding CI job](https://circleci.com/gh/iterative/dvc.org/992?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link):

![image](https://user-images.githubusercontent.com/1477535/66226025-de004f80-e6a7-11e9-9435-185fe1e860d9.png)

But maybe we should test committing badly formatted code...

Reply:
Wait actually why did it find 0 changed files? That's not right 🙁 ...

Reply:
Oh wait maybe that's OK because there are no JS or MD file changes in that commit 😋

Reply:
Hmmm. This isn't working actually. See https://circleci.com/gh/iterative/dvc.org/1000. Any thoughts on how to setup `pretty-quick` correctly here @iAdramelk?

- [ ] Remove test commit be661b5 from this branch before trying to merge PR...

Reply:
@jorgeorpinel should you specify a key to force it run the check on all files (vs only changed - as it seems that's what it's trying to do - and thus there are 0 of them)

Reply:
When I run `yarn pretty-quick --check --verbose` on this branch locally, even with a clean Git working tree, it finds 11 changed files. I'm not sure what it's comparing it to. Maybe master? But there's 13 files changed in this PR so. Actually it's pretty mysterious... I wonder how the CI sets up the git repo actually, let me check.

Reply:
The CI seems to setup the git repo correctly, I even tried replicating how it checks out code, but I still see 12 changed files when running `yarn format-check`, while CircleCI sees 0. 🤷‍♂

Reply:
👀

Reply:
@jorgeorpinel added an extra step to fix this

Reply:
Thank you!